pkgs/by-name/rh/rhsrvany/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rhsrvany";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rwmjones";
+    repo = "rhsrvany";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-eeEiAdm7NO5fFYKtHQbeBq4RhP8Xwgw2p2Wkm+n0EWM=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  configureFlags = [ "--bindir=${placeholder "out"}/share/" ];
```

Comment:
Why put binaries in share/ (instead of bin/)?

Reply:
Perhaps put the rationale in a code comment?

Reply:
Fedora and Debian have it under `/usr/share/virt-tools/` so I followed that, but they probably have it so that no random Windows binaries are in the path.
For the usecase within nixpkgs and `virt-v2v` will eitherway just have a symlink to the dir with the exe binaries.